components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtil.java

```diff
@@ -185,6 +172,12 @@ public static void makeAsyncAPICall(SecurityEventTokenPayload securityEventToken
             }
             requestBuilder.setBody(jsonString, ContentType.APPLICATION_JSON);
         } catch (IOException | IdentityEventException | ParseException e) {
+            if(e instanceof IdentityEventException) {
+                if (((IdentityEventException) e).getErrorCode().equals(ERROR_RETRIEVING_ENCRYPTION_PUBLIC_KEY.getCode())) {
```

Comment:
Shall we make this other way around?
`ERROR_RETRIEVING_ENCRYPTION_PUBLIC_KEY.getCode().equals(((IdentityEventException) e).getErrorCode())`